### ISSUE TITLE
Data reader

### ISSUE THREAD

**Issue by bryanlimy** [bryanlimy/V1T#2]
- Extract and read data from [https://gin.g-node.org/cajal/Sensorium2022](https://gin.g-node.org/cajal/Sensorium2022)
- Create PyTorch DataLoader.
- Separate a held-out test set from the pre-training data that we shall never look at until the end of the submission for final testing.
